sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuWindowExec.scala

```diff
@@ -481,7 +481,7 @@ object GpuWindowExec {
   def isBatchedRunningFunc(func: Expression, spec: GpuWindowSpecDefinition): Boolean = {
     val isSpecOkay = isRunningWindow(spec)
     val isFuncOkay = func match {
-      case _: GpuBatchedRunningWindowWithFixer => true
+      case f: GpuBatchedRunningWindowWithFixer => f.newFixer().isDefined
```

Comment:
I'm not sure I like creating a fixer to see if it is defined. Just because this runs on the driver. Not that any of them do anything that needs a GPU right now. But mostly to avoid that situation.

Reply:
Right. I toyed with adding to the interface, and picked the wrong route. I'll change this.